ncm-ceph/src/main/perl/Ceph/ClusterMap.pm

```diff
@@ -0,0 +1,231 @@
+#${PMpre} NCM::Component::Ceph::ClusterMap${PMpost}
+
+use 5.10.1;
+
+use parent qw(CAF::Object);
+use Readonly;
+use Data::Dumper;
+use JSON::XS;
+
+
+sub _initialize
+{
+    my ($self, $clusterobj) = @_;
+
+    $self->{log} = $clusterobj;
+
+    $self->{quattor} = {};
+    $self->{ceph} = {};
+
+    $self->{Cluster} = $clusterobj;
```

Comment:
why capital C?

Reply:
cluster is int the schema, and is set as attribute of the Cluster class. so to make the destinction i used capital here. could rename this to clusterobj if that is prefered